config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
I think we should either use `config/initializers/content_security_policy.rb` or remove it

Reply:
As it is all commented out and it is a ?default rails file? I have left it in, I found on SE that these files were best kept in when you do an upgrade to rails